test/parallel/test-buffer-isencoding.js

```diff
@@ -17,7 +17,8 @@ const assert = require('assert');
     assert.strictEqual(Buffer.isEncoding(enc), true);
   });
 
-[ 'utf9',
+[ '',
+  'utf9',
```

Comment:
I also think that after line 35 the encoding should actually be used (perhaps `Buffer.from()`?), its important that the `isEncoding()` should agree with actual behaviour of APIs that use encodings, and its not clear to me ATM that it does, so I think the assert should ensure consitency.